ivi-layermanagement-examples/LayerManagerControl/src/Expression.cpp

```diff
@@ -215,6 +215,12 @@ ExpressionList Expression::getNextExpressionClosure(string text)
 
             if (expr->isVar())
             {
+                if((expr->mName).compare("<file>") &&
+                   (expr->mName).compare("<filename>"))
+                {
+                    transform(text.begin(), text.end(), text.begin(), ::tolower);
+                }
```

Comment:
You can remove it outside of if statement. you are also calling transform() in else statement. Then, you don't need to call transform function in two different places